test/lit/downgrade-reftypes.wast

```diff
@@ -0,0 +1,61 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py and should not be edited.
+
+;; Write to a binary, lowering away refined GC types.
+;; RUN: wasm-as %s -all --disable-gc -g -o %s.wasm
+
+;; Read it back and verify that the types were lowered away.
+;; RUN: wasm-dis %s.wasm -all -o - | filecheck %s
```

Comment:
You can just do `wasm-opt --roundtrip` for this kind of thing.

Reply:
Though maybe as it is is more explicit? I don't feel strongly.

Reply:
In particular I want to write without GC enabled and then read with GC enabled. I don't think --roundtrip would let me do that.

Reply:
Ah, good point...